bazel/README.md

```diff
@@ -571,21 +571,31 @@ test/run_envoy_bazel_coverage.sh
 The summary results are printed to the standard output and the full coverage
 report is available in `generated/coverage/coverage.html`.
 
+To generate coverage results for fuzz targets, use the `FUZZ_COVERAGE` environment variable, e.g.:
+```
+FUZZ_COVERAGE=true VALIDATE_COVERAGE=false test/run_envoy_bazel_coverage.sh
+```
+This generates a coverage report for fuzz targets after running the target for one minute against fuzzing engine libfuzzer using its coprus as initial seed inputs. The full coverage report will be available in `generated/fuzz_coverage/coverage.html`.
+
 Coverage for every PR is available in Circle in the "artifacts" tab of the coverage job. You will
 need to navigate down and open "coverage.html" but then you can navigate per normal. NOTE: We
 have seen some issues with seeing the artifacts tab. If you can't see it, log out of Circle, and
 then log back in and it should start working.
 
 The latest coverage report for master is available
-[here](https://storage.googleapis.com/envoy-coverage/report-master/index.html).
+[here](https://storage.googleapis.com/envoy-coverage/report-master/index.html). The latest fuzz coverage report for master is available [here](https://storage.googleapis.com/envoy-fuzz-coverage/report-master/index.html)
```

Comment:
merge master, this should be in https://storage.googleapis.com/envoy-postsubmit/master/fuzz_coverage/index.html